classification/scripts/models.py

```diff
@@ -0,0 +1,191 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision import models
+
+
+# --------------------
+# Utility Modules
+# --------------------
+
+class InitialConv(nn.Module):
+    def __init__(self, in_channels, out_channels):
+        super().__init__()
+        self.conv = nn.Conv2d(in_channels, out_channels, kernel_size=7,
```

Comment:
The kernel size and stride are hard coded according to paper?
I'd rather have them as parameters initialised with default values.